src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,18 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/user-service-hw?serverTimeZone=UTC</property>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">1234qwer</property>
+        <property name="show_sql">false</property>
+        <property name="format_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.Movie"/>
+        <mapping class="mate.academy.model.MovieSession"/>
+        <mapping class="mate.academy.model.CinemaHall"/>
+        <mapping class="mate.academy.model.User"/>
+
+    </session-factory>
+</hibernate-configuration>
```

Comment:
Leave one empty line at the end of the file

Reply:
not fixed. add blank line at the EOF